cadwyn/structure/versions.py

```diff
@@ -448,7 +448,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
```

Comment:
We need an inner `if` instead. In your scenario, Cadwyn will change the logic of returning null-responses from the default logic of FastAPI which is going to cause a lot of undefined behavior.

Reply:
no, its not, `None` in FastAPI will result in `b''`, not `b"null"`